rllib/env/policy_client.py

```diff
@@ -268,6 +283,7 @@ def run(self):
                     time.sleep(999999)
 
             return ExternalEnvWrapper(real_env)
+        return real_env
 
```

Comment:
Is there a test we can extend to cover these missing cases?

Reply:
I'll look into it.